fvm/bootstrap.go

```diff
@@ -396,6 +396,8 @@ func (b *bootstrapExecutor) Execute() error {
 		b.mintInitialTokens(service, fungibleToken, flowToken, b.initialTokenSupply)
 	}
 
+	b.deployExecutionParameters(fungibleToken, &env)
+	b.setupExecutionWeights(fungibleToken)
```

Comment:
@janezpodhostnik is this the correct way that I should be deploying and initializing these values? I figured we wanted the `FlowExecutuionParameters` smart contract to be on a different account than the `FlowServiceAccount` smart contract to make it similar to what is on testnet and mainnet, so I'm deploying it to the fungible token account. 

Then I change `setupExecutionWeights` to use the `fungibleToken` account instead of the service account and deploy the service account after since it relies on those parameters.

When I run the tests though, there are a lot of failing tests as if the weights were not set properly or are not being retrieved properly. I'm not sure if I am missing something

Reply:
Thank you very much for doing this.

> is this the correct way that I should be deploying and initializing these values? 
Yes! the `fungibleToken` account is a good choice!

The failing tests are probably because you need to change the `AccountNameExecutionParametersAccount: executionParametersAccountFunc,` in `fvm/systemcontracts/system_contracts.go` so that the FVM know to look on the `fungibleToken` account to get the weights

Reply:
Thanks for the help! I just updated that and it looks like the tests are passing now, so this is ready for review. 🙂